frontend/src/v5/ui/routes/dashboard/projects/tickets/ticketsTable/ticketsTable.component.tsx

```diff
@@ -164,12 +164,14 @@ export const TicketsTable = ({ isNewTicketDirty, setTicketValue }: TicketsTableP
 	
 	const [presetFilters, setPresetFilters] = useState<TicketFilter[]>(); 
 	useEffect(() => {
+		TicketsCardActionsDispatchers.setFiltering(true)
```

Comment:
Instead of using redux can you pass the filtering to TicketsFiltersContextComponent?
Or did you try this before an encountered problems?